### PR TITLE
Pages UI

### PR DESCRIPTION
**Pull request by LiamANeeson**
Added Styling to pages on the website. Added dummy text and placeholders for images that will be added. Website needs to be populated with more content.

### CODE REVIEW

File: client/src/Pages/Community/Community.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.community-container {
```
Comment (alex-mulkerrins) [inline client/src/Pages/Community/Community.css:1]: Do we have standardised colour Palette?

Comment (LiamANeeson) [reply client/src/Pages/Community/Community.css:1]: Yes but not included anywhere in the documentation currently. Some colors are for placeholders (see aqua blue). I will add palette to README.

File: client/src/Pages/Community/Community.js [hunk 2]
```diff
@@ -1,8 +1,75 @@
 import React from 'react'
+import './Community.css'
 
 function Community() {
   return (
-    <div>Community</div>
+    <div className='community-container'>
```
Comment (alex-mulkerrins) [inline client/src/Pages/Community/Community.js:6]: I think this section looks good in general as a static layout, calling the css etc..., I imagine we can begin to problem solve a dynamic layout adding likes and reply's after this

File: client/src/Pages/Home/Home.js [hunk 3]
```diff
@@ -17,13 +17,13 @@ function Home() {
         <h2 className='box-head'>Box 1</h2>
```
Comment (alex-mulkerrins) [inline client/src/Pages/Home/Home.js:17]: I think all looks good here

File: client/src/Pages/Community/Community.css (hunk 1 above)
Comment (alex-mulkerrins) [reply client/src/Pages/Community/Community.css:1]: Ok sounds good, everything else here I think looks good, was there any other line of this PR you were unsure of?

File: client/src/Pages/Community/Community.js (hunk 2 above)
Comment (alex-mulkerrins) [reply client/src/Pages/Community/Community.js:6]: sweet, sounds good

**Comment by PrajaktaKulkarni26** [comment]
This is for UI development of pages - User Authentication

**Pull request merged**